### ISSUE TITLE
Export/import  Product on Nopcomarce 3.8

### ISSUE THREAD

**Issue by ofekpc** [nopSolutions/nopCommerce#1665]
there is change on the way assign filed of category on product between v 3.7 to 3.8
as i see the logical to do this change is wrong
on v3.7 the import and export of product per category based on categoryid
withch is good , since  this filed is unique
but on version 3.8 you change it based on category witch is not unique 
so for example if I have category call ps3->game   and ps4->
so when i import and put in the filed  "game" how it is know  witch category to go the ps3 or the ps4
if was category id it can work
in short the CategoryID is uniq the category is not
on the nop 3.8 beta it was per categoryid

**Comment by AndreiMaz** [comment]
it's by design. name we use names
